mne/dipole.py

```diff
@@ -166,10 +219,25 @@ def plot_locations(self, trans, subject, subjects_dir=None,
             The path to the freesurfer subjects reconstructions.
             It corresponds to Freesurfer environment variable SUBJECTS_DIR.
             The default is None.
+        bgcolor : tuple of length 3
+            Background color in 3D.
+        opacity : float in [0, 1]
+            Opacity of brain mesh.
+        brain_color : tuple of length 3
+            Brain color.
+        fig_name : str
+            Mayavi figure name.
+        fig_size : tuple of length 2
+            Mayavi figure size.
```

Comment:
The conditional on line 292 can also be removed, since ``plot_dipole_locations`` no longer supports plotting with mayavi.